src/Microsoft.DocAsCode.Common/Git/GitUtility.cs

```diff
@@ -13,6 +13,7 @@ namespace Microsoft.DocAsCode.Common.Git;
 
 public static class GitUtility
 {
+    private const string GitTimeoutEnvVarName = "DOVFX_GIT_TIMEOUT";
     private static readonly string CommandName = "git";
     private static readonly int GitTimeOut = 1000;
```

Comment:
Maybe we can also relax the default 1s a little bit to 10 seconds.

Reply:
Updated